liquidhaskell-boot/src/Language/Haskell/Liquid/Bare.hs

```diff
@@ -254,7 +260,7 @@ makeGhcSpec0 cfg ghcTyLookupEnv tcg instEnvs lenv localVars src lmap targetSpec
                     | v <- gsReflects refl
                     ]
                 , dataDecls = Bare.dataDeclSize mySpec $ dataDecls mySpec
-                , measures  = mconcat $ map Ms.measures $ mySpec : map snd dependencySpecs
+                , measures  = mconcat $ map Ms.measures $ map snd dependencySpecs ++ [mySpec]
```

Comment:
Maybe this also requires a small comment, lest someone changes this back (for performance).